bofire/benchmarks/benchmark.py

```diff
@@ -87,11 +111,15 @@ def _single_run(
         X = X[benchmark.domain.inputs.get_keys()]
         Y = benchmark.f(X)
         XY = pd.concat([X, Y], axis=1)
+        # pd.concat() changes datatype of str to np.int32 if column contains whole numbers.
+        # colum needs to be converted back to str to be added to the benchmark domain.
         strategy.tell(XY)
         metric_values[i] = metric(strategy.domain)
         pbar.set_description(
             f"run {run_idx:02d} with current best {metric_values[i]:0.3f}"
         )
+        if (i + 1) % 1 == 0:
```

Comment:
Why this, this just means that you want to log every iteartion, why then the if clause and not just `autosage_results`? From my perspective, we should add there a logging parameter which specifies how often we log as additonal argument. Can you implement it?